### DISCUSSION TITLE
sudo: usbip: command not found

### DISCUSSION THREAD

**Discussion by Max1274** [dorssel/usbipd-win#130] [Q&A] [answered]
I'm running Windows 11 and Docker Desktop on WSL2 and I'm receiving the following error when trying to attach USB device

```
usbipd wsl attach -b 1-1
[sudo] password for Ubuntu:
sudo: usbip: command not found
usbipd: error: Failed to attach device with ID '1-1'.
```

What's the problem here? 
Thanks in advance! :)

**Comment by dorssel** [comment, marked as answer]
Your problem: `usbip: command not found`.
Have you followed the instructions on https://github.com/dorssel/usbipd-win/wiki/WSL-support to setup the USBIP client software on the WSL end of things?
